src/rpc/rpc.cc

```diff
@@ -588,6 +588,46 @@ namespace rpc {
       return it->second;
   }
 
+  struct request_frame {
```

Comment:
could you please add a couple words in the commit message explaining the motivation of this change?

Reply:
That's to make request_frame declared before `future<> client::request(...)` for the next patch. Will add